CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 # FL Accelerator Changelog
 
+##  January 2023 release
+
+We are excited to announce the release of the November iteration of our [FL Accelerator repository](https://github.com/Azure-Samples/azure-ml-federated-learning).
+
+Here below we list all the new features. The most significant changes are the introduction of a guide to help onboard people to FL on Azure ML, the addition of a tutorial explaining how to implement _Differential Privacy_, the support of third-party FL frameworks (_NVFlare_ and _Flower_), and the support of distributed training.
+
+### FL Experience
+- Introduced support for third-party FL frameworks. The [pneumonia example](./docs/real-world-examples/pneumonia.md) can now be run using _NVFlare_ (instructions [here](./docs/frameworks/nvflare.md)) or _Flower_ (instructions [there](./docs/frameworks/flower.md)). 
```

Comment:
proposition:

- Experimental support for third-party FL frameworks... (rest)

nit: instead of "instructions here" you could just say `see [tutorial](./docs/...)` ?